SPECS/libstoragemgmt/libstoragemgmt.spec

```diff
@@ -192,12 +192,15 @@ install -m 755 tools/udev/scan-scsi-target \
     %{buildroot}/%{_udevrulesdir}/../scan-scsi-target
 
 %check
+check_status=0
 if ! make check
 then
-  cat test-suite.log || true
-  exit 1
+  cat test-suite.log
+  check_status=1
 fi
 
+[[ $check_status -eq 0 ]]
```

Comment:
Neat.:)

Reply:
Giving credit where credit is due.
Thank you @PawelWMS for teaching me this trick :)